OpenRA.Game/Graphics/SpriteFont.cs

```diff
@@ -123,8 +123,8 @@ public void DrawText(string text, float2 location, Color c)
 				// Convert screen coordinates back to UI coordinates for drawing
 				if (g.Sprite != null)
 					Game.Renderer.RgbaSpriteRenderer.DrawSprite(g.Sprite,
-					(screen + g.Offset).ToFloat2() / deviceScale,
-					g.Sprite.Size / deviceScale);
+						(screen + g.Offset).ToFloat2() / deviceScale,
+						g.Sprite.Size / deviceScale);
```

Comment:
Please revert this white-space change. Even though it does look somewhat better :man_shrugging: Also I see a lot of double-tab indentations elsewhere in your code. I'm not sure what are our conventions for multi-line formatting :thinking: Someone with an opinion should step up here.

Reply:
Opinionated person chiming in :smile: 
As far as I'm aware, this formatting change is correct. I use tab on line continuation all the time and never gotten flak for it. I think it is a good convention to follow.

Double-tab is definitely gratuitous, but this may be just an editor issue regarding 4-space tabs vs 8-space tabs and not a deliberate thing.